rpgJavaInterpreter-core/src/main/kotlin/com/smeup/rpgparser/parsing/parsetreetoast/mute.kt

```diff
@@ -23,17 +23,21 @@ data class MuteAnnotationExecutionLogEntry(override val programName: String, val
 }
 
 fun MuteParser.MuteLineContext.toAst(conf: ToAstConfiguration = ToAstConfiguration(), position: Position? = null): MuteAnnotation {
-    val annotation = this.muteAnnotation()
-    return when (annotation) {
-        is MuteParser.MuteComparisonAnnotationContext -> {
-            val errors = LinkedList<Error>()
-            val val1 = RpgParserFacade().createParser(BOMInputStream(("".padStart(8) + annotation.val1.text.substring(1, annotation.val1.text.lastIndex)).byteInputStream(Charsets.UTF_8)), errors, longLines = true)
-                    .expression()
-                    .toAst(conf)
+    fun extractExpressionFrom(token: Token): Expression {
+        return RpgParserFacade().createParser(
+            BOMInputStream(
+                ("".padStart(8) + token.text.substring(
+                    1,
+                    token.text.lastIndex
+                )).byteInputStream(Charsets.UTF_8)
+            ), LinkedList(), longLines = true
```

Comment:
```suggestion
            ), errors = mutableListOf(), longLines = true
```

This could be slightly more obvious